fieldservice_account/security/ir.model.access.csv

```diff
@@ -1,2 +1,6 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_account_invoice_line_fsm_user,account.invoice.line.fsm.user,model_account_invoice_line,fieldservice.group_fsm_user,1,1,1,1
+access_account_invoice_line_portal,access.account.invoice.line.portal,account.model_account_invoice_line,base.group_portal,1,0,0,0
```

Comment:
This is already in [odoo accounting](https://github.com/odoo/odoo/blob/12.0/addons/account/security/ir.model.access.csv)

Reply:
I [confirm](https://github.com/odoo/odoo/blob/12.0/addons/account/security/ir.model.access.csv#L43) that. @bodedra ?

Reply:
Good catch @brian10048  This changes are redundant.